### PR TITLE
Add skills, links and styles for disabled buttons

### PR DESCRIPTION
**Pull request by peoxia**

### CODE REVIEW

File: jared/index.html [hunk 1]
```diff
@@ -102,6 +102,8 @@ <h2 class="ds-heading">Core Skills</h2>
                <li>UX Design</li>
                <li>UX Research</li>
                <li>Figma</li>
+               <li>Wireframing</li>
+               <li>Prototyping</li>
                <li>Presentations</li>
                <li>Project planning</li>
                <li>Business strategy</li>
```
Comment (JaredGrobler) [inline jared/index.html:109]: The Business strategy would best be placed in the other skills section 👍

File: jared/assets/css/main.css [hunk 2]
```diff
@@ -66,6 +66,9 @@ svg,img { max-width: 100%; height: auto; }
 
 .ds-button{ display: inline-block; text-transform: uppercase;font-size: 12px;line-height: 13px;font-weight: 600;letter-spacing: 3px; color:#F26464; padding:17px 30px;  border: 2px solid #F26464;border-radius: 60px; }
 .ds-button:hover{ background-color:#F26464 ; color: #FFF; }
+.ds-button-disabled{ display: inline-block; text-transform: uppercase;font-size: 12px;line-height: 13px;font-weight: 600;letter-spacing: 3px; color:gray; padding:17px 30px;  border: 2px solid gray;border-radius: 60px; }
+.ds-button-disabled:hover{ color: gray; }
+.ds-arrow-button i{ display: inline-block; vertical-align: middle; }
```
Comment (JaredGrobler) [inline jared/assets/css/main.css:71]: This makes sense as some projects will need to be edited for a while and hence disabled. Would it be possible to adjust the alpha levels to a slightly lower opacity?

File: jared/index.html [hunk 3]
```diff
@@ -185,8 +187,8 @@ <h3 class="ds-work-tilte">U-X.design</h3>
                       <section>
                           <h3 class="ds-work-tilte">FineDine</h3>
                           <p>A fictional home catering application inspiredby current food delivery applications.</p>
-                          <a href="works-setails.html" class="ds-button">Case study</a>
-                          <a href="works-setails.html" class="ds-button">Figma file</a>
+                          <a href="javascript:;" class="ds-button-disabled">Case study</a>
+                          <a href="https://www.figma.com/file/0SgQKj5qmWTSRFLTziLz1t/High-end-Delivery" class="ds-button">Figma file</a>
```
Comment (JaredGrobler) [inline jared/index.html:191]: Thanks

File: jared/assets/css/main.css (hunk 2 above)
Comment (peoxia) [reply jared/assets/css/main.css:71]: I understood you want the button to be more lighter gray. Still curious what "alpha levels" mean :joy:

File: jared/index.html (hunk 1 above)
Comment (peoxia) [reply jared/index.html:109]: :heavy_check_mark:

File: jared/assets/css/main.css (hunk 2 above)
Comment (peoxia) [reply jared/assets/css/main.css]: Check if `#bbbbbb` color works better, it's in preview now

Comment (peoxia) [reply jared/assets/css/main.css]: I've also changed cursor to be default instead of hand pointer for these buttons

**Pull request merged**